contracts/Token.sol

```diff
@@ -1,22 +1,22 @@
 pragma solidity ^0.5.0;
 
 import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
-import "openzeppelin-solidity/contracts/token/ERC20/ERC20Detailed.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 
-contract Token is ERC20, ERC20Detailed {
+contract Token is Ownable, ERC20 {
   uint40 public startTime;
   uint40 public endTime;
 
-  mapping(address => Token) public tokens;
-  constructor(uint40 _startTime, uint40 _endTime, uint256 initialSupply) ERC20Detailed("Gold", "GLD", 18) public {
+  constructor(uint40 _startTime, uint40 _endTime, uint256 initialSupply) public {
```

Comment:
@laurenamy why did you get rid of the ERC20Detailed contract btw?

Reply:
It wasn't really serving a purpose and I didn't see a reason to include it in the future issues, but I can add it back in if necessary!